### PR TITLE
[TACHYON-1197] Enable findbugs for calling await() without a loop

### PR DESCRIPTION
**Pull request by aaudiber**
https://tachyon.atlassian.net/browse/TACHYON-1197

### CODE REVIEW

**Comment by yupeng9** [comment]
LGTM. Thanks for adding this check and improving `ScheduledTimer`!

File: common/src/main/java/tachyon/heartbeat/ScheduledTimer.java [hunk 1]
```diff
@@ -62,6 +65,10 @@ public String getThreadName() {
   protected void schedule() {
     mLock.lock();
     try {
+      if (mScheduled) {
+        throw new IllegalStateException("Called schedule twice without waiting for any ticks");
+      }
```
Comment (calvinjia) [inline common/src/main/java/tachyon/heartbeat/ScheduledTimer.java]: Could `preconditions` be used here?

Comment (aaudiber) [inline common/src/main/java/tachyon/heartbeat/ScheduledTimer.java]: Good call - changed, thanks

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**